starter-code/index.html

```diff
@@ -1,42 +1,88 @@
 <!DOCTYPE html>
 <html lang="en">
+    <title>spotify</title>
+    <link rel="stylesheet" type="text/css" href="../starter-code/css/main.css">
+
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+  <nav class="navbar">
+    <span href ="www.spotify.com"><img class="spotify" src="./images/spotify-logo.png" alt="logo-spotify"></img></span> 
+    <span href ="www.spotify.com">Premium</span> 
+    <span href ="www.spotify.com">Discover</span>
+    <span href ="www.spotify.com">Help</span>
+    <span href ="www.spotify.com">Download</span>
+  </nav>
+  
+  <div class="concert">
+    <h1>Music for everyone.</h1>
+    <h2>Spotify is now free on mobile, tablet and computer.</h2>
+    <h2> Listen to the right music, wherever you are.</h2>
+  </div>
+ 
+  <div class="col">
+
+    <h3>What’s on Spotify?</h3>
+
+    <colones class="col col-3">
+      <img src="./images/music-icon.png" class="logo"></img>
+      <h4>Millions of Songs</h4>
+      <p>There are millions of songs on Spotify</p>
+    </colones>
+
+    <colones class="col col-3">
+      <img src="./images/high-quality-icon.png" class="logo"></img>
+      <h4>HD Music</h4>
+      <p>Listen to music as if you were listening live</p>
+    </colones>
+    
+    <colones class="col col-3">
+      <img src="./images/devices-icon.png" class="logo"></img>
+      <h4>Stream Everywhere</h4>
+      <p>Stream music on your smartphone, tablet or computer</p>
+    </colones>
+
+  </div>
+  
+  <div class="container">
+    <h5>It’s as yeezy as Kanye West.</h5>
+     
+    <section class="container section left">
+          <h6>Search</h6>
+          <p>
+            <em>Know what you want to listen to?</em>
+            <em>Just search and hit play.</em>
+          </p>
+
+          <h6>Browse</h6>
+          <p>
+            <em>Check out the latest charts,</em>
```

Comment:
You shouldn't use em like this, you should use p tag instead.
This em tag is supposed to highlight piece of text in a big text per exemple.

try stay simple and use only what you need.